openedx/frontend-app-learning
# feat: show an alert when relative dates are due

# Description

There used to be an alert on the unit page that informed users about the missed deadline and allowed them to reset the due dates. When we implemented enforcing these dates, we had to hide this banner, as there were lots of code dependencies tied to this. We still want to warn students that the unit can't be modified in an alert.

### Note: Implementation depends on [this PR](https://github.com/openedx/edx-platform/pull/34290)

# Testing Steps

1. Checkout [this branch](https://github.com/openedx/edx-platform/pull/34290) in edx-platform
2.  Add a deadline to any assigment or exam
3. Navigate to it as a non-staff learner
4. Assert the banner appears.

# Screenshots

![image](https://github.com/openedx/frontend-app-learning/assets/19278837/c443a43d-1702-4127-9f9f-a81b57ccc705)
